src/deephaven_ib/__init__.py

```diff
@@ -150,7 +150,8 @@ class BarDataType(Enum):
     """Bid/Ask yield."""
     YIELD_LAST = 14
     """Last yield."""
-
+    AGGTRADES = 15
+    """Aggregate trade prices."""
```

Comment:
If this is a new type, it should appear in `example_all_functionality.py`, so that we can quickly run a script to confirm that it works.